packages/story-editor/src/components/canvas/framesLayer.js

```diff
@@ -114,14 +132,10 @@ function FramesLayer() {
           onContextMenu={onOpenMenu}
           onScroll={onScroll}
         >
-          {currentPage &&
-            currentPage.elements.map((element) => {
-              return <FrameElement key={element.id} element={element} />;
-            })}
-          <DesignSpaceGuideline
-            ref={setDesignSpaceGuideline}
-            isVisible={isAnythingTransforming}
-          />
+          {elementIds.map((id) => {
+            return <FrameElement key={id} id={id} />;
```

Comment:
nit: typically we try to avoid keys being ids, no?

Reply:
We want to avoid using _array indexes_ for keys, but using the unique element ID here is fine